src/managers/GuildMemberManager.js

```diff
@@ -191,6 +236,39 @@ class GuildMemberManager extends BaseManager {
     return endpoint.post({ data: body, reason }).then(data => data.pruned);
   }
 
+  /**
+   * Kicks a user from the guild.
+   * <info>The user must be a member of the guild</info>
+   * @param {UserResolvable} user The member to kick
+   * @param {string} [reason] Reason for kicking
+   * @returns {Promise<GuildMember|User|Snowflake>} Result object will be resolved as specifically as possible.
+   * If the GuildMember cannot be resolved, the User will instead be attempted to be resolved. If that also cannot
+   * be resolved, the user ID will be the result.
+   * @example
+   * // Kick a user by ID (or with a user/guild member object)
+   * guild.members.kick('84484653687267328')
+   *   .then(user => console.log(`Kicked ${user.username || user.id || user} from ${guild.name}`))
+   *   .catch(console.error);
+   */
+  kick(user, reason) {
+    const id = this.client.users.resolveID(user);
+    if (!id) return Promise.reject(new TypeError('INVALID_TYPE', 'user', 'UserResolvable'));
+
+    return this.client.api
+      .guilds(this.guild.id)
+      .members(id)
+      .delete({ reason })
+      .then(() => {
+        if (user instanceof GuildMember) return user;
+        const _user = this.client.users.resolve(id);
+        if (_user) {
+          const member = this.resolve(_user);
+          return member || _user;
```

Comment:
```suggestion
          return this.resolve(_user) || _user;
```
seems a bit pointless making a variable here since its just going to be returned on the next line,